staging/src/k8s.io/client-go/rest/factory.go

```diff
@@ -97,6 +98,9 @@ func (r RESTClientFactory) NewClientWithOptions(apiPath string, config ClientCon
 		// per client values
 		versionedAPIPath: DefaultVersionedAPIPath(apiPath, config.GroupVersion),
 		content:          config,
+		// defaults
+		rateLimiter:      flowcontrol.NewTokenBucketRateLimiter(DefaultQPS, DefaultBurst),
```

Comment:
it is defaulted on RESTClientFor this way, but I don't have clear if this should be here or in the clientset,